### PR TITLE
GEOD-435 - Add Radio Interferences

### PR DESCRIPTION
**Pull request by oehm-smith**
* Tested that it loads and saves correctly

### CODE REVIEW

File: src/client/app/radio-interference/radio-interference-group.component.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <div class="panel-body" [hidden]="! isGroupOpen">
+        <div *ngIf="isEmptyCollection()">
+            No information available for {{getItemName()}}s
+        </div>
+        <div formArrayName="radioInterferences">
+            <div *ngFor="let radioInterference of getItemsCollection(); let i = index">
+                <!-- [geodesyEvent] - send event to child components -->
+                <!-- (open/close Above/Below Event) - receive events from child components -->
+                <!-- (returnEvents) - single point to call events on parent -->
+                <radio-interference-item [groupArray]="groupArrayForm"
```
Comment (frankfu03) [inline src/client/app/radio-interference/radio-interference-group.component.html]: You may remove the 3 commented lines.

**Comment by frankfu03** [comment]
:+1:

File: src/client/app/radio-interference/radio-interference-item.component.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+    <div class="panel-body" [hidden]="!isOpen">
+        <div *ngIf=" getItem() == null ">
+            No information available for current {{getItemName()}}
+        </div>
+        <div *ngIf="getItem() != null" [ngClass]="{'container-dirty': getItem().dateDeleted}">
+            <div class="form-group">
+                <text-input formControlName="possibleProblemSource" controlName="possibleProblemSource" [form]="itemGroup">Possible Problem Source</text-input>
+                <text-input formControlName="observedDegradation" controlName="observedDegradation" [form]="itemGroup">Observed Degradation</text-input>
+                <datetime-input [form]="itemGroup" controlName="startDate">Date Installed (UTC)</datetime-input>
```
Comment (lbodor) [inline src/client/app/radio-interference/radio-interference-item.component.html]: It's not date installed, but start date.

File: src/client/app/radio-interference/radio-interference-item.component.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                <datetime-input [form]="itemGroup" controlName="endDate" [required]="index != 0">Date Removed (UTC)</datetime-input>
```
Comment (lbodor) [inline src/client/app/radio-interference/radio-interference-item.component.html]: It's end date, not date removed.

**Pull request merged**